blueprints/validator/files/app/validators/__name__.js

```diff
@@ -1,7 +1,22 @@
 import BaseValidator from 'ember-cp-validations/validators/base';
 
-export default BaseValidator.extend({
+const <%= classifiedModuleName %> = BaseValidator.extend({
   validate(/*value, options, model, attribute*/) {
     return true;
   }
 });
+
+<%= classifiedModuleName %>.reopenClass({
+  /**
+   * Define attribute specific dependent keys for your validator
+   *
+   * @param {String}  attribute   The attribute being evaluated
+   * @param {Unknown} options     Options passed into your validator
+   * @return {Array}
+   */
+  getDependentsFor(/* attribute, options */) {
+    return [];
+  }
+});
+
+export default <%= classifiedModuleName %>;
```

Comment:
@blimmer Thoughts about this new default validator template? Does it feel too complicated? I decided to expose this so anyone creating a new validator will know its there.